config/cookstyle.yml

```diff
@@ -1,16 +1,467 @@
 AllCops:
+  DisabledByDefault: true
```

Comment:
this isn't working for me over in chefstyle, since chefstyle.yml is part of the "default" config this just kinda seems to get ignored.